README.md

```diff
@@ -14,11 +14,12 @@
 ### CSS snippet
 
 - reduced motion - adds a reduced motion snippet with prefix `transition`
+- dark-color-scheme - adds dark prefers-color-scheme snippet with prefix `darkColorScheme`
+- light-color-scheme - adds light prefers-color-scheme snippet with prefix `lightColorScheme`
+
 🚨 Safari Only Queries  🚨 
 - less-contrast - adds less contrast media snippet with prefix `less-contrast`
 
-- dark-color-scheme - adds dark prefers-color-scheme snippet with prefix `darkColorScheme`
```

Comment:
I moved this up because its actually [supported by all browsers expect IE](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme#browser_compatibility).

Reply:
⭐